pkg/steps/build_client.go

```diff
@@ -0,0 +1,41 @@
+package steps
+
+import (
+	"io"
+
+	"k8s.io/client-go/rest"
+
+	buildapi "github.com/openshift/api/build/v1"
+	"github.com/openshift/client-go/build/clientset/versioned/scheme"
+	buildclientset "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
+)
+
+type BuildClient interface {
+	buildclientset.BuildInterface
+	Logs(name string, options *buildapi.BuildLogOptions) (io.ReadCloser, error)
```

Comment:
Should be in a `BuildsExtension` on the `BuildInterface` upstream 

/cc @bparees

Reply:
Is there a precedent for that?  I don't see such a thing on AppsInterface (deploymentconfig) either and these are generated clients.  Logs aren't really part of the resource itself.

Reply:
That is how logs for Pods are handled in k8s client-go